torch/nn/functional.py

```diff
@@ -2473,6 +2478,14 @@ def interpolate(input, size=None, scale_factor=None, mode='nearest', align_corne
         See :class:`~torch.nn.Upsample` for concrete examples on how this
         affects the outputs.
 
+        When scale_factor are specified, if use_scale_factor=False,
+        scale_factor is used to compute the output_size which will then
+        be used to infer new scales for the interpolation. This is the current
```

Comment:
This reads like implementation detail to me: do we really need to tell users about this? I think what users should know about is how including this parameter changes the ONNX graph.

Reply:
@zou3519, it is important to explain the difference in behavior since this parameter also affects PyTorch's interpolate output, to use the correct values of the scales when use_scale_factor=True, as explained in this message.

Reply:
I'm having trouble, from reading this description, understanding how this parameter affects the interpolate output. But maybe that is because I don't understand the interpolate operator.

Reply:
resolving based on https://github.com/pytorch/pytorch/pull/28324#discussion_r355661922